pypykatz/lsadecryptor/packages/msv/decryptor.py

```diff
@@ -358,6 +367,23 @@ def add_credman_credential(self, credman_credential_entry):
 			
 		self.current_logonsession.credman_creds.append(c)
 		
+	def get_proxy_info(self, msv1_0_module):
+		old_pos = self.reader.tell()
+
+		msv1_0_module = self.reader.reader.get_module_by_name("msv1_0.dll")
+
+		try:
+			position = self.find_signature('msv1_0.dll', b'\xe0zRE}*\xecL\xb2\x14s\x9fAY\xc3\x92')
```

Comment:
How many LSASS dumps/live machine has this signature been tested against?
Is this always static? I'd hazard to guess it's not.

Reply:
On Win 10 19H1 18362 and Win 11 23H2 22631 it matched, but I'm not sure if it generally holds for all Windows versions using credential guard.